### ISSUE TITLE
Effect.replicate appears under 'utils' instead of 'Collecting'

### ISSUE THREAD

**Issue by dmeehan1968** [Effect-TS/effect#4074]
### What is the issue?

Effect.replicate is out of sequence in the TOC, appearing under the 'utils' category.

Docstrings in the source seem to be correct (although the 'Category' is shown as 'collecting & elements' but this is the same as other functions which are appearing under the 'Collecting' category.

### Where did you find it?

https://effect-ts.github.io/effect/effect/Effect.ts.html#replicate

**Comment by jessekelly881** [comment]
Does `Effect.replicate` even need to exist when there is the more general `Array.replicate`?

**Comment by gcanti** [comment]
> Does Effect.replicate even need to exist when there is the more general Array.replicate?

Yeah, historically this function may have predated `Array.replicate` and has remained there without us noticing, but it's unnecessary. 

In any case, it's a simple utility and doesn't belong in the "Collecting" category.

**Comment by mikearnaldi** [comment]
> Does `Effect.replicate` even need to exist when there is the more general `Array.replicate`?

no it doesn't, it's there due to history most likely
